### DISCUSSION TITLE
Can't enable ONLYOFFICE

### DISCUSSION THREAD

**Discussion by zeitue** [nextcloud/all-in-one#1367] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Added `ONLYOFFICE_ENABLED=yes` to compose file
2. Added `ONLYOFFICE_SECRET=4132585b70bbbeb77487599eaad60` to compose file
3. Started the `docker-compose.yml` file
4. Went to https://localhost:8080
5. Logged in to the AIO login
6. Set domain name

### Expected behavior 
To see ONLYOFFICE in the list of options

### Actual behavior 
No ONLYOFFICE, only Collabora, Talk, ClamAV, Imaginary,  and Fulltextsearch options.

### Host OS 
Ubuntu 22.04

#### Nextcloud AIO version 
Nextcloud AIO v2.1.1

#### Current channel 
Latest

#### Other valuable info 
I went ahead and enabled all the options and started the setup and after finishing there is still no ONLYOFFICE container.

**Comment by Zoey2936** [comment, marked as answer]
OnlyOffice got removed see: https://github.com/nextcloud/all-in-one/discussions/1031
